src/components/DetailsOverlay/SearchRangeInput/index.tsx

```diff
@@ -0,0 +1,72 @@
+import styled from "@emotion/styled";
+import TextInput from "@leafygreen-ui/text-input";
+import { QueryParams } from "constants/queryParams";
+import { size } from "constants/tokens";
+import { useQueryParam } from "hooks/useQueryParam";
+import { FilterRow, StyledSubtitle } from "../styles";
+
+const SearchRangeInput: React.FC = () => {
+  const [lowerBound, setLowerQueryParam] = useQueryParam<number | undefined>(
+    QueryParams.LowerRange,
+    undefined
+  );
+  const [upperBound, setUpperBound] = useQueryParam<number | undefined>(
+    QueryParams.UpperRange,
+    undefined
+  );
+
+  const updateLowerBound = (newVal: string) => {
+    if (newVal === "") {
+      setLowerQueryParam(undefined);
+    } else {
+      setLowerQueryParam(parseInt(newVal, 10));
+    }
+  };
+
+  const updateUpperBound = (newVal: string) => {
+    if (newVal === "") {
+      setUpperBound(undefined);
+    } else {
+      setUpperBound(parseInt(newVal, 10));
+    }
```

Comment:
This also doesn't prevent against upperBound being less than lowerBound.
<img width="676" alt="image" src="https://user-images.githubusercontent.com/4605522/190700203-5454a404-bd37-440a-a8cb-6cd8ba539e20.png">